src/auth/utils.js

```diff
@@ -31,10 +29,10 @@ const getUrlParts = (url) => {
   }
 };
 
-const logFrontendAuthError = (error) => {
+const logFrontendAuthError = (loggingService, error) => {
```

Comment:
As mentioned elsewhere, this is a small compromise for simplicity - we pass the loggingService through here so that this function can stay as a stand-alone helper.  Otherwise it'd need to be attached to an instance so it could do `this.loggingService`, and we also need it in multiple classes.